crates/services/p2p/src/request_response/messages.rs

```diff
@@ -75,6 +79,7 @@ pub enum NetworkResponse {
 #[derive(Debug, Clone)]
 pub enum OutboundResponse {
     Block(Option<Arc<SealedBlock>>),
+    BlocksInclusive(Vec<SealedBlock>),
     SealedHeader(Option<Arc<SealedBlockHeader>>),
     Transactions(Option<Arc<Vec<Transaction>>>),
```

Comment:
Sorry for the wrong description of the task. The initial idea was to support range requests for headers and transactions.

Reply:
This PR is still just doing headers. It might make sense to do transactions in a separate PR at this point?

Reply:
Of course, if it makes development easier=)